README.md

```diff
@@ -7,24 +7,25 @@ This project contains Rust extensions to speed up the [official Python driver fo
 
 
 ## Installation
-Adjust your `requirements.txt` or similar the line `neo4j` with:
+Adjust your dependencies (`requirements.txt`, `pyproject.toml` or similar) like so:
 ```
 # remove:
 # neo4j == 5.X.Y
 # add:
 neo4j-rust-ext ~= 5.X.Y.0
 ```
 
+I.e., install the same version of `neo4j-rust-ext` as you would install of `neo4j` (except for the last segment which is used for patches of this library).  
 That's it!
-You don't have to change your code.
+You don't have to change your code but can use the driver as you normally would.
 This package will install the driver as its dependency and then inject itself in a place where the driver can find it and pick it up.
```

Comment:
Should we point out the minimum version of neo4j-rust-ext somewhere in this doc?

Reply:
Good idea. Added.